server/src/api/tickets/ticket.service.ts

```diff
@@ -32,14 +80,68 @@ export const toTicket = (row:any): Ticket => {
   };
 };
 
-export const reduceToTicketState = (res: any, t: Ticket) => {
+const reduceToTicketState = (res: any, t: Ticket) => {
   const id = t.event_instance_id;
   const {byId, allIds} = res;
   return allIds.includes(id) ?
     res :
     {byId: {...byId, [id]: t}, allIds: [...allIds, id]};
 };
 
+
+//
```

Comment:
Not a blocker but could add a short description, the function names are pretty descriptive though